src/WCSimPMTObject.cc

```diff
@@ -3300,47 +3301,95 @@ G4double* PMT3inchR14374::Getqpe()
 
 }
 
-// TF: Used WebPlotDigitizer on Fig.2a from VLVnT13 proceedings, red curve (average)
-G4double* PMT3inchR14374::GetQEWavelength(){
-
-  //TEST: make QE same!!
-  //static G4double wavelength_value[20] = { 280., 300., 320., 340., 360., 380., 400., 420., 440., 460., 480., 500., 520., 540., 560., 580., 600., 620., 640., 660.};
-  static G4double wavelength_value[21] = { 300., 320., 340., 360., 380., 400., 420., 440., 460., 480., 500., 520., 540., 560., 580., 600., 620., 640., 660., 680., 700.};
-  return wavelength_value;
+#if 0 // from ETEL
```

Comment:
I prefer defining new PMT objects instead of using `#if`, so that we can easily change the PMT type through macro instead of recompiling the program